### ISSUE TITLE
Assertions on external tasks

### ISSUE THREAD

**Issue by ingorichtsmeier** [camunda/camunda-bpm-assert#112]
As a user I would like to write:
```
asssertThat(processInstance).isWaitingAt("MyExternalTask").externalTask().hasTopic("my-external-task-topic");
complete(externalTask(), withVariables("var1", value1, "var2", value2);
```

**Comment by zambrovski** [comment]
Will implement it on a fork.

**Comment by zambrovski** [comment]
Just released a version 2.1 containing this feature.

**Comment by tmetzke** [comment]
Hi Ingo,

thanks for your examples!
We have included this in the 3.0.0-alpha1 release.
Feel free to test the new external task features.
Every feedback is appreciated of course.

Closing this as fixed.

Best regards,
Tobias
